src/app/(teacher)/majors/page.module.css

```
@@ -0,0 +1,330 @@
+.majorList section {
+  gap: 0;
+}
+
+.majorList section > h2 {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  padding: 0;
+  margin: -1px;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)